model/cloud.go

```diff
@@ -84,3 +84,27 @@ type Subscription struct {
 	DNS        string   `json:"dns"`
 	IsPaidTier string   `json:"is_paid_tier"`
 }
+
+// Invoice model represents a cloud invoice
+type Invoice struct {
+	ID             string             `json:"id"`
+	Number         string             `json:"number"`
+	CreateAt       int64              `json:"create_at"`
+	Total          int64              `json:"total"`
```

Comment:
Are we missing the `Tax` field that we included in CWS?

Reply:
Added along with the Type for line item